src/SimpleWebRequest.ts

```diff
@@ -123,16 +123,16 @@ export interface WebRequestOptions {
     augmentErrorResponse?: (resp: WebErrorResponse) => void;
 }
 
-function isJsonContentType(ct: string) {
-    return ct && ct.indexOf('application/json') === 0;
+function isJsonContentType(ct: string): boolean {
+    return ct.indexOf('application/json') === 0;
 }
 
-function isFormContentType(ct: string) {
-    return ct && ct.indexOf('application/x-www-form-urlencoded') === 0;
```

Comment:
This isn't related to eslint/tslint - I'd assume the falsey check was here for a reason

Reply:
@berickson1 I added return type `boolean` to fix linter warnings because `ct` is `string`, and function expects only string, maybe there is a case where it receives not a string, we can add `string | undefined`; I can revert this change, and add conditions to check if value is not Falsy.

Reply:
I think the least invasive change is below
`return !!ct && ct.indexOf('application/x-www-form-urlencoded') === 0;`

Reply:
@berickson1 oke.